`apoc.meta.stats` for 4.1.0.8 fails whereas 4.1.0.3 succeeds when a rel is back ticked

_[reported by a customer]_

Neo4j Version:   4.1.3
APOC Version: 4.1.0.8

```cypher
create (n:Class {id:1})-[r:`http://www.w3.org/2000/01/rdf-schema#isDefinedBy`]->(s:Student);
call apoc.meta.stats();
```

results in 

```cypher
@neo4j> call apoc.meta.stats();
Failed to invoke procedure `apoc.meta.stats`: Caused by: org.neo4j.exceptions.SyntaxException: Invalid input ':': expected an identifier character, whitespace, '|', a length specification, a property map or ']' (line 1, column 23 (offset: 22))
"MATCH (:Class)-[r:http://www.w3.org/2000/01/rdf-schema#isDefinedBy]->() RETURN count(r) AS count"
```

--- a/core/src/main/java/org/neo4j/cypher/export/DatabaseSubGraph.java
+++ b/core/src/main/java/org/neo4j/cypher/export/DatabaseSubGraph.java
@@ -11,6 +11,7 @@ import org.neo4j.graphdb.schema.IndexDefinition;
 import java.util.Iterator;
 
 import static apoc.export.cypher.formatter.CypherFormatterUtils.cypherNode;
+import static apoc.util.Util.quote;
 
 public class DatabaseSubGraph implements SubGraph
 {
@@ -85,7 +86,7 @@ public class DatabaseSubGraph implements SubGraph
     public long countsForRelationship(Label start, RelationshipType type, Label end) {
         String startNode = cypherNode(start);
         String endNode = cypherNode(end);
-        String relationship = String.format("[r:%s]", type.name());
+        String relationship = String.format("[r:%s]", quote(type.name()));
         return transaction.execute(String.format("MATCH %s-%s->%s RETURN count(r) AS count", startNode, relationship, endNode))
                 .<Long>columnAs("count")
                 .next();
@@ -93,7 +94,7 @@ public class DatabaseSubGraph implements SubGraph
 
     @Override
     public long countsForNode(Label label) {
-        return transaction.execute(String.format("MATCH (n:%s) RETURN count(n) AS count", label.name()))
+        return transaction.execute(String.format("MATCH (n:%s) RETURN count(n) AS count", quote(label.name())))
                 .<Long>columnAs("count")
                 .next();
     }

--- a/core/src/test/java/apoc/meta/MetaTest.java
+++ b/core/src/test/java/apoc/meta/MetaTest.java
@@ -1073,4 +1073,20 @@ public class MetaTest {
                         "RETURN *",
                 assertResult);
     }
+    
+    @Test
+    public void testMetaStatsWithTwoDots() {
+        db.executeTransactionally("CREATE (n:`My:Label` {id:1})-[r:`http://www.w3.org/2000/01/rdf-schema#isDefinedBy` {alpha: 'beta'}]->(s:Another)");
+        TestUtil.testCall(db, "CALL apoc.meta.stats()", row -> {
+            assertEquals(map("My:Label", 1L, "Another", 1L), row.get("labels"));
+            assertEquals(2L, row.get("labelCount"));
+            assertEquals(map("http://www.w3.org/2000/01/rdf-schema#isDefinedBy", 1L), row.get("relTypesCount"));
+            assertEquals(2L, row.get("propertyKeyCount"));
+            assertEquals(map("()-[:http://www.w3.org/2000/01/rdf-schema#isDefinedBy]->(:Another)", 1L,
+                    "()-[:http://www.w3.org/2000/01/rdf-schema#isDefinedBy]->()", 1L,
+                    "(:My:Label)-[:http://www.w3.org/2000/01/rdf-schema#isDefinedBy]->()",1L),
+                    row.get("relTypes"));
+        });
+
+    }
 }

